code-styling/markdown/README.md

```diff
@@ -0,0 +1,1155 @@
+  - [5.3. Task lists](#53-task-lists)
+
+- [6. Horizontal rules](#6-horizontal-rules)
+
+- [7. Code](#7-code)
+
+  - [7.1. Inline code](#71-inline-code)
+  - [7.2. Code blocks](#72-code-blocks)
+
+- [8. Links](#8-links)
+
+  - [8.1. Inline links](#81-inline-links)
+  - [8.2. Reference links](#82-reference-links)
+  - [8.3. Automatic links](#83-automatic-links)
+  - [8.4. Linking to books, blog and other external documents](#84-linking-to-books-blogs-and-other-external-documents)
+
+- [9. Images](#9-images)
+
+- [10. Tables](#10-tables)
+
+- [11. Emoji](#11-emoji)
+
+- [12. HTML](#12-html)
+
+## 1. Basics
+
+## 1.1. Filenames
+
+- All Markdown files MUST have the `.md` file extension.
+
+- It is RECOMMENDED to base filenames on the top-header level title.
+
+- Filenames MUST be lowercase, unless it is one of the following:
+  - `README.md`
+  - `LICENSE.md`
+  - `CONTRIBUTING.md`
+  - `CHANGELOG.md`
+
+- Filenames MUST NOT start with articles (e.g. `the`, `a`, `an`).
+
+- Filenames MUST NOT contain special characters, it is RECOMMENDED to replace them with hyphens.
+
+- Filenames MUST NOT contain spaces and/or punctuation characters, it is RECOMMENDED to replace them with hyphens.
+
+- Filenames MUST NOT start or end with a hyphen.
+
+- Filenames MUST NOT have consecutive hyphens.
+
+Good filenames:
+
+```
+markdown-style-guide.md
+introduction-into-coding.md
+hello-world.md
+```
+
+Bad filenames:
+
+```
+.markdown-style.guide.md
+-introduction-into-coding.md
+the-hello-world-example.md
+double--hypens.md
+```
+
+### 1.2 Files
+
+- Files MUST use the Unix LF (line feed) line ending.
+
+- Files MUST only use UTF-8 encoding without BOM.
+
+- Files MUST end with a single blank line.
+
+- Lines SHOULD NOT be longer than 120 characters; lines longer SHOULD be split into multiple subsequent lines.
+
+- It is RECOMMENDED to break large paragraphs logically when trying to keep lines under 120 characters:
+  <!--alex ignore period-->
+  - _sentences_: after a period (`.`), question (`?`) or exclamation mark (`!`)
+  - _clauses_: after words like `and`, `which`, `if` ... `then`, commas (`,`)
+
+- Lines SHOULD NOT be split into multiple line using word-breaks.
+
+- Consecutive blank lines MUST NOT exists, with the exception of code blocks where the code style guide for the coding 
+  language applies.
+
+- There MUST NOT be trailing whitespace at the end of non-blank lines unless
+  it has a function such as indicating a line break.
+
+- In the rare case a line break is absolutely needed, one SHALL end the line with exactly two spaces.
+
+- There MUST NOT be trailing whitespace at the end of blank lines.
+
+- Lines MUST use an indent of 2 spaces and MUST NOT use tabs for indenting.
+
+- A single space MUST be used after sentences, when followed by another sentence.
+
+- A file MUST start with a header on the first line.
+
+### 1.3 Language
+
+<!--alex ignore american-->
+- The language MUST be English, and in particular American English.
```

Comment:
This is about our guidelines, so we decide ;)
Since the IT GDP in the USA is way bigger, than the UK, this seem the way to go.
Also, most of the online documentation takes en-us as the default.